HTTP.cabal

```diff
@@ -1,7 +1,7 @@
 Cabal-Version: >= 1.10
 Name: HTTP
 Version: 4000.4.1
-x-revision: 3
+x-revision: 4
```

Comment:
actually, I think this bump should only happen when uploading to hackage? Not 100% sure.

Reply:
Well, the plan would be to push this revision to Hackage.

Dry run:
Pushing "HTTP.cabal" (HTTP-4000.4.1~4) [review-mode] ...
Hackage response was (after 0.298 secs):

Changes:

<ul>
  
 * Changed tested-with
                          from `ghc ==9.8.0: ghc ==9.6.3: ghc ==9.4.7: ghc ==9.2.8: ghc ==9.0.2: ghc ==8.10.7: ghc ==8.8.4: ghc ==8.6.5: ghc ==8.4.4: ghc ==8.2.2: ghc ==8.0.2: ghc ==7.10.3`
                          to `ghc ==9.10.0: ghc ==9.8.2: ghc ==9.6.4: ghc ==9.4.8: ghc ==9.2.8: ghc ==9.0.2: ghc ==8.10.7: ghc ==8.8.4: ghc ==8.6.5: ghc ==8.4.4: ghc ==8.2.2: ghc ==8.0.2`

 * Changed the library component&#39;s library dependency on &#39;base&#39;
                          from `>=4.6.0.0 && <4.20`
                          to `>=4.6.0.0 && <4.21`

 * Changed the library component&#39;s library dependency on &#39;network&#39;
                          from `>=2.4 && <3.2`
                          to `>=2.4 && <3.3`

 * Changed the library component&#39;s library dependency on &#39;time&#39;
                          from `>=1.1.2.3 && <1.13`
                          to `>=1.1.2.3 && <1.15`

</ul>